ci/create_database.py

```diff
@@ -25,13 +26,24 @@ def generate_token(size=12):
         f.write(json.dumps(config))
 
     with open(f'sql-config.cnf', 'w') as f:
-        f.write(f'''
-[client]
+        f.write(f'''[client]
 host={config["host"]}
 user={config["user"]}
 password="{config["password"]}"
 database={config["db"]}
 ''')
+        if 'ssl-ca' in config:
+            f.write(f'ssl-ca={config["ssl-ca"]}\n')
+        if 'ssl-cert' in config:
+            f.write(f'ssl-cert={config["ssl-cert"]}\n')
+        if 'ssl-key' in config:
+            f.write(f'ssl-key={config["ssl-key"]}\n')
+        if 'ssl-mode' in config:
+            f.write(f'ssl-mode={config["ssl-mode"]}\n')
+
+    shutil.copy('/sql-config/server-ca.pem', 'server-ca.pem')
```

Comment:
What happens if these files don't exist? Will this be breaking right now? Why are the options above optional, but you're assuming these files exist.